tendrl/commons/jobs/__init__.py

```diff
@@ -68,10 +68,10 @@ def _run(self):
                             status=raw_job['status'],
                             payload=json.dumps(raw_job['payload']),
                             errors=raw_job['errors']).save()
-
-                        raw_job['payload']['parameters']['integration_id'] = \
-                            raw_job[
-                            'payload']['integration_id']
+                        if "integration_id" in raw_job['payload']:
+                            raw_job['payload']['parameters']['integration_id'] = \
+                                raw_job[
+                                'payload']['integration_id']
```

Comment:
Is there any impact on message or job updates if payload['integration_id'] is not present?